spec/models/usa_city_spec.rb

```diff
@@ -0,0 +1,14 @@
+require 'rails_helper'
+
+RSpec.describe UsaCity, type: :model do
+  let (:city1) {
+    UsaCity.create(
+                   name: "Denver",
+               state_id: 14
```

Comment:
Can you nudge this over so the alignment looks better. Sorry, I know that's super nitpicky :cactus: